Source/Manager/ClientConfiguration.swift

```diff
@@ -68,21 +68,36 @@ public struct ClientConfiguration {
         /// payment.schibsted.no
         case norway
 
-        fileprivate func loadConfiguration() -> (serverURL: URL, trackingProvider: String?) {
+        fileprivate static func loadConfigurationData() -> [String: [String: String]] {
```

Comment:
Does this need to be static? Or could it just be a lazy var or something?
static vars is created on app startup, so the more static vars the slower the app launch.